experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -453,15 +453,15 @@ describe("FormAudience", () => {
       <Subject
         experiment={{
           ...MOCK_EXPERIMENT,
-          application: NimbusExperimentApplicationEnum.DESKTOP,
+          application: NimbusExperimentApplicationEnum.FENIX,
           channel: NimbusExperimentChannelEnum.NIGHTLY,
-          targetingConfigSlug: "WIN_ONLY",
+          targetingConfigSlug: "MOBILE_ONLY",
           targetingConfig: [
             {
-              label: "Win Only",
```

Comment:
since you have removed all the windows targeting to mobile, I would suggest adding a test with windows